helm/solr-operator/Chart.yaml

```diff
@@ -95,4 +95,10 @@ annotations:
             name: "example"
         numThreads: 4
         image:
-          tag: 8.7.0
\ No newline at end of file
+          tag: 8.7.0
+
+dependencies:
+  - name: 'zookeeper-operator'
+    version: 0.2.9
+    repository: https://charts.pravega.io
+    condition: useZkOperator
```

Comment:
We probably want to change this to a different variable, such as `zookeeper-operator.enabled`, because we want users to be able to use the zookeeper-operator, without installing it as a dependency. That requires two variables. Then when setting the command line variable for solr, we can do an OR of `zookeeper-operator.enabled` and `useZkOperator`.

I'm flexible on names, if you don't think that's clear.

Reply:
Tell me what you think of the names I have chosen.

Reply:
[The helm docs](https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags) give some good guidance. I think this logic is very good in general. We should just use the name `zookeeper-operator` instead of `zookeeperOperator`, since that's the name of the dependency.

We should also think about maybe including backwards compatibility of the old `useZookeeperOperator` option, which was unfortunately a string `"true"`.

I can take a stab at updating the documentation around this.